.github/workflows/wheel-builder.yml

```diff
@@ -295,6 +308,7 @@ jobs:
         PYTHON:
           - {VERSION: "3.11", "ABI_VERSION": "py37"}
           - {VERSION: "3.11", "ABI_VERSION": "py39"}
```

Comment:
Can py39 be removed per https://github.com/pyca/cryptography/issues/12854#issuecomment-2849394290?

Reply:
Same here